Tests/Unit/Controller/NewsBaseControllerTest.php

```diff
@@ -61,7 +51,7 @@ public function NoNewsFoundConfigurationRedirectsToListView()
             ['redirect']);
         $mock->expects($this->once())
             ->method('redirect')->with('list');
-        $mock->_call('handleNoNewsFoundError', 'redirectToListView');
+        $mock->_call('handleNoNewsFoundError', array('detail' => array('errorHandling' => 'redirectToListView')));
```

Comment:
This and all following instances of `array()` should be switched to shorthand syntax `[]`.